Readme.md

```diff
@@ -97,89 +106,89 @@ const $person: Codec<Person> = $.record(
 Hovering over the error squigglies will reveal the following diagnostic.
 
 ```
-Type 'Record<[["name", Codec<string>], ["nickName", Codec<string>]], "name" | "nickName", Codec<string>>' is not assignable to type 'Codec<Person>'.
-  The types returned by '_d(...)' are incompatible between these types.
-    Property 'superPower' is missing in type '{ name: string; } & { nickName: string; }' but required in type 'Person'.
+Type 'Codec<{ nickName: string; superPower: string | undefined; }>' is not assignable to type 'Codec<Person>'.
+  The types returned by 'decode(...)' are incompatible between these types.
+    Type '{ nickName: string; superPower: string | undefined; }' is not assignable to type 'Person'.
 ```
 
-## Error Handling
-
-This library **intentionally** does not check for conditions that would suggest an error. If an error is produced, it is because of invalid input or incompatibility between the input and codec definition. This library prioritizes performance over end-developer DX, as it is not intended for end developers, but rather for tool developers.
```

Comment:
@harrysolovay Is this deletion correct, in light of #2?

Reply:
👍